package.json

```diff
@@ -16,17 +16,17 @@
     "prepublishOnly": "npm run build",
     "start": "fedx-scripts webpack-dev-server --progress",
     "debug-test": "node --inspect-brk node_modules/.bin/jest --coverage --runInBand",
-    "test": "TZ=GMT fedx-scripts jest --coverage --runInBand",
+    "test": "TZ=GMT fedx-scripts jest --coverage --maxWorkers=2",
     "snapshot": "TZ=GMT fedx-scripts jest --updateSnapshot",
     "stylelint": "stylelint '**/*.scss'"
   },
   "license": "AGPL-3.0",
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
-    "@edx/frontend-component-footer": "npm:@edx/frontend-component-footer-edx@^6.4.0", 
-    "@edx/frontend-platform": "5.6.1",
+    "@edx/frontend-component-footer": "npm:@edx/frontend-component-footer-edx@^7.1.0",
+    "@edx/frontend-platform": "^8.0.0",
     "@edx/openedx-atlas": "^0.6.0",
-    "@edx/paragon": "^20.44.0",
+    "@openedx/paragon": "^21.11.3",
```

Comment:
Why are we moving some packages from the edx namespace to openedx?

Reply:
You can view [axim-engineering#23](https://github.com/openedx/axim-engineering/issues/23) for details. I have also updated the description